api/src/pcapi/routes/adage/v1/serialization/prebooking.py

```diff
@@ -17,6 +17,12 @@
 from pcapi.serialization.utils import to_camel
 
 
+class BookingsInstitutionQueryModel(AdageBaseResponseModel):
+    source: str
+    destination: str
+    ids: list[int]
```

Comment:
Des noms plus explicites pourraient aider la compréhension. Suggestion : 

```suggestion
    source_uai: str
    destination_uai: str
    booking_ids: list[int]
```